typescript/agentkit/README.md

```diff
@@ -543,6 +544,49 @@ const walletProvider = new ViemWalletProvider(client, {
 });
 ```
 
+### PrivyWalletProvider
```

Comment:
reminder to add the new sections to the table of contents at the top of the file!